files/image_config/procdockerstats/procdockerstats

```diff
@@ -0,0 +1,155 @@
+# !/usr/bin/env python
+'''
+This code is for a specific daemon, process-docker.
+Which sends process and docker CPU/memory utilization data to DB every 2 mins.
+'''
+
+import sys, errno
+import os
+import time
+import syslog
+import signal
+import select
+import subprocess
+import re
+import swsssdk 
+import collections
+from datetime import datetime
+
+VERSION = '1.0'
+
+SYSLOG_IDENTIFIER = "process-docker"
+
+REDIS_HOSTIP = "127.0.0.1"
+
+# ========================== Syslog wrappers ==========================
+def log_info(msg, also_print_to_console=False):
+    syslog.openlog(SYSLOG_IDENTIFIER)
+    syslog.syslog(syslog.LOG_INFO, msg)
+    syslog.closelog()
+
+def log_warning(msg, also_print_to_console=False):
+    syslog.openlog(SYSLOG_IDENTIFIER)
+    syslog.syslog(syslog.LOG_WARNING, msg)
+    syslog.closelog()
+
+def log_error(msg, also_print_to_console=False):
+    syslog.openlog(SYSLOG_IDENTIFIER)
+    syslog.syslog(syslog.LOG_ERR, msg)
+    syslog.closelog()
+
+# ========================== ProcessDocker class ==========================
+class ProcDockerStats:
+
+    def __init__(self):
+        self.swid = 0
+        self.running = False
+        self.handle = None
+        
+    def initialize(self):
+        self.state_db = swsssdk.SonicV2Connector(host=REDIS_HOSTIP)
+        self.state_db.connect("STATE_DB")
+        # wipe out all data before updating with new values
+        self.state_db.delete_all_by_pattern('STATE_DB', 'Docker_Stats|*')
+              
+    def run_commands(self, commands):
```

Comment:
Looks like you've repurposed this function from an existing daemon, and have modified it slightly. I think it makes more sense in this daemon if it only took a single command as a parameter, not a list, since it is now returning command output. It's probably better to separate that output (process vs. docker)

Reply:
separated logic. added method for docker. similarly I will add for process stats